### DISCUSSION TITLE
Get state of staking

### DISCUSSION THREAD

**Discussion by HinsonSIDAN** [Anastasia-Labs/single-asset-staking-offchain#12] [Q&A]
Building the endpoint of getting state of staking, so my question is:

What are the necessary fields and how to get them?

**Comment by nikhils9** [comment]
I missed adding the endpoint to fetch staking state. I'll work on getting them added. Raised an issue https://github.com/Anastasia-Labs/single-asset-staking-offchain/issues/14

**Reply by HinsonSIDAN** [reply]
Please let me know if there is any updates, thanks!!
